packages/loader/container-loader/src/deltaManagerProxy.ts

```diff
@@ -52,29 +49,19 @@ export class DeltaQueueProxy<T> extends EventForwarder<IDeltaQueueEvents<T>> imp
     }
 
     public async systemPause(): Promise<void> {
-        this.systemPaused = true;
-        return this.queue.pause();
+        return this.pause();
     }
```

Comment:
@curtisman - (not that you changed this behavior, just makes me curious to read this code)

Why don't we plumb through systemPause -> systemPause? This must be an important feature of the `DeltaQueueProxy`, but it's pretty strange at face value. A caller using `IDeltaQueue` won't know whether `systemPause` is going to hit the real `DeltaQueue` as a system pause or a user pause...

Reply:
Wait actually maybe your change did do something... It looks like the real DeltaQueue logic for system v. user pause is the same as what you deleted here - just keeping track of two different (in name only) reasons to pause, and only resume if both are un-paused.

If someone says this:
- systemPause()
- pause()
- systemResume()

against an `IDeltaQueue` that is actually a `DeltaQueueProxy`, I think it will resume... no?  But before your change it would have not.

Or... never mind this hits the new ref-count implementation and is ok ^_^  In which case get rid of the distinction in the APIs (i.e. remove `systemPause`), right?